assets/lambda/api-handlers/analysis-pipeline/queue_analysis_pipeline_run.py

```diff
@@ -6,32 +6,13 @@
 
 import boto3
 from botocore.exceptions import ClientError
+from capepy.aws.utils import decode_error
 
 logger = logging.getLogger(__name__)
 
 sqs_client = boto3.client("sqs")
```

Comment:
In a perfect world. The current capepy isn't as fully featured as it should be. This can either be something we do some planning and architecting before this is merged or we can do it after